harness/determined/layers/_worker_process.py

```diff
@@ -60,6 +60,11 @@ def __init__(self, broadcast_client: ipc.ZMQBroadcastClient):
         # checks on it.
         self._broadcast_client.send(ipc.ConnectedMessage(process_id=os.getpid()))
 
+        # Avoid hangs when receiving from the broadcast server.  Run this after sending the
+        # ConnectedMessage so that the broadcast server has a pid for running a healthcheck while
+        # this runs.
+        self._broadcast_client.safe_start()
```

Comment:
q: is this path only used to synchronize workloads? Do we communicate workloads that are only generated on the local chief? Sorry I forgot we had concluded. Can we include this answer on why we need a broadcast client in the comment?

Reply:
On tip-of-master, the workload information flows from the master, through each local chief, to each local worker.  There is no such thing as "workloads generated on the local chief".

I'd rather not change the message because honestly, this call and the corresponding comment has already been deleted further downstream (when workloads originate from the chief worker rather than from the master).